zend_abstract_interface/config/config.c

```diff
@@ -140,11 +140,75 @@ void zai_config_mshutdown(void) {
 void zai_config_runtime_config_ctor(void);
 void zai_config_runtime_config_dtor(void);
 
-void zai_config_first_time_rinit(void) {
+#if PHP_VERSION_ID < 70300
+#define GC_ADD_FLAGS(c, flag) GC_FLAGS(c) |= flag
+#define GC_ADDREF(p) ++GC_REFCOUNT(p)
+#endif
+
+static void zai_config_intern_zval(zval *pzval) {
+    if (Z_TYPE_P(pzval) == IS_STRING) {
+#if PHP_VERSION_ID >= 70400
+        ZVAL_INTERNED_STR(pzval, zend_new_interned_string(Z_STR_P(pzval)));
```

Comment:
During a request, `zend_new_interned_string` will store strings in an arena in `CG(interned_strings)` (well, at least on PHP 8.3). Is that long enough for the values being store during `zai_config_first_time_rinit`? I worry that it's not.

Reply:
Ah, I missed that. Thanks!